src/lib/components/settings/Settings.svelte

```diff
@@ -4,30 +4,49 @@
 	import { Modal, Separator, Switch } from '~/lib/components';
 	import { Github, Gitlab } from '~/lib/icons';
 	import LogOutButton from './LogOutButton.svelte';
-	import { onMount } from 'svelte';
+	import { onDestroy, onMount } from 'svelte';
 	import { settings } from '~/lib/stores';
 	import { storage } from '~/lib/helpers';
 	import type { Settings } from '~/lib/types';
+	import { browser } from '$app/environment';
 
 	let user = $page.data.session?.user;
 	let mounted = false;
+	let forceOpenSettings = false;
 
 	const options: Array<Settings['notificationAxis']> = ['Auto', 'Vertical', 'Horizontal'];
 
+	function handleKeyDown(event: KeyboardEvent) {
+		if (browser && event.keyCode === 188 && event.metaKey) {
+			event.preventDefault();
+			forceOpenSettings = true;
+		} else {
+			forceOpenSettings = false;
```

Comment:
I don't think we want to close the modal when pressing any other key

Reply:
How can we force again to open the modal when using the keybind, closing the modal, and using the keybind again? I'm not too familiar with Svelte so that was the only way I found to trigger again the modal.

Reply:
[df18af6](https://github.com/colinlienard/gitlight/pull/74/commits/df18af60d50d47204345c158dbb4f2dce0c35f34) By binding the state to the component, it will be able to mutate the state:
```svelte
<Modal title="Settings" bind:open={forceOpenSettings}>
```

Reply:
Nice!